src/android/app/src/main/java/io/github/lime3ds/android/features/settings/ui/SettingsAdapter.kt

```diff
@@ -258,16 +261,33 @@ class SettingsAdapter(
         val sliderBinding = DialogSliderBinding.inflate(inflater)
 
         textSliderValue = sliderBinding.textValue
-        textSliderValue!!.text = sliderProgress.toString()
+        textSliderValue!!.setText(sliderProgress.toString())
         sliderBinding.textUnits.text = item.units
 
         sliderBinding.slider.apply {
             valueFrom = item.min.toFloat()
             valueTo = item.max.toFloat()
             value = sliderProgress.toFloat()
+            textSliderValue!!.addTextChangedListener( object : TextWatcher {
+                    override fun afterTextChanged(s: Editable) {
+                        val textValue = s.toString().toIntOrNull();
+                        if (textValue == null || textValue < valueFrom || textValue > valueTo) {
+                            textSliderValue!!.setTextColor(Color.parseColor("#ff0000"));
+                        } else {
+                            textSliderValue!!.setTextColor(Color.parseColor("#000000"));
```

Comment:
This code does not account for devices using dark mode.

Reply:
Good catch. Any guesses as to where the theme color settings are stored? If you don't know off the top of your head I can try to search for it, no worries.

Reply:
Figured this one out. Ended up removing it completely and resolving by using the official Material Design spec for errors, as seen in the screenshot below. Let me know any thoughts.

![Screenshot_20240728_144226_Lime3DS](https://github.com/user-attachments/assets/4f14e5b1-28d2-46f2-81e7-44eae1d29918)